content/doc/upgrade-guide/2.60.adoc

```diff
@@ -0,0 +1,53 @@
+---
+layout: documentation
+title:  Jenkins Upgrade Guide
+notitle: true
+---
+
+== Upgrading to Jenkins LTS 2.60.x
+
+Each section covers the upgrade from the previous LTS release, the section on 2.60.1 covers the upgrade from 2.46.3.
+
+=== Upgrading to Jenkins LTS 2.60.1
+
+==== Java 8 Required
+
+Jenkins 2.60.1 is the first LTS release to require Java 8.
```

Comment:
@MarkEWaite I don't think it's necessary. It's the first item in the list, there's the explicit title, and users are likely to get here from the changelog at https://jenkins.io/changelog-stable/ and the original comment has a screenshot how that will look like for 2.60.1 with the big yellow box on top and the bold item in the list. At some point, there's diminishing returns, and I think we reached it. Sure, some people will miss this, but I don't think we'll get them by making a word in the upgrade guide (they're not reading anyway) bold.